en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md

```diff
@@ -0,0 +1,720 @@
+# BigQuery Connector Example
+
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
+
```

Comment:
Currently it supports only read only operations. It is assumed a third party system is using BigQuery and the Big query connector is used to query information.

Reply:
if true, better mention it ahead.